packages/adapter-cloudflare-workers/index.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@cloudflare/workers-types" />
```

Comment:
cc: @gtm-nayan do you still think we should omit the worker types reference here to avoid polluting the ambient namespace?

Reply:
Personally yes, but I should note that my usage of Cloudflare is only limited to a couple of projects so we could get more empirical.

Are the declarations limited to types or do they also add globals?

Reply:
I'm not entirely sure myself. @dario-piotrowicz what do you think about this? Are there any other frameworks doing this?

As an aside, I'm already getting them as ambient types because I have the drizzle orm d1 driver imported and they're doing this.